docs/source/bft_configuration.md

```diff
@@ -0,0 +1,228 @@
+      Port: 7050
+      MSPID: OrdererOrg2
+      Identity: /path/to/identity
+      ClientTLSCert: path/to/ClientTLSCert1
+      ServerTLSCert: path/to/ServerTLSCert1
+    - ID: 3
+      Host: bft2.example.com
+      Port: 7050
+      MSPID: OrdererOrg3
+      Identity: /path/to/identity
+      ClientTLSCert: path/to/ClientTLSCert2
+      ServerTLSCert: path/to/ServerTLSCert2
+    - ID: 4
+      Host: bft3.example.com
+      Port: 7050
+      MSPID: OrdererOrg4
+      Identity: /path/to/identity
+      ClientTLSCert: path/to/ClientTLSCert3
+      ServerTLSCert: path/to/ServerTLSCert3
+```
+
+When the channel config block is created, the `configtxgen` tool reads the paths
+to the identities and TLS certificates, and replaces the paths with the corresponding bytes of
+the identities and certificates.
+
+Note: it is possible to remove or add an ordering node to a channel dynamically, a process described in the Reconfiguration section below.
+
+In addition, note that the [channel capabilities](./capabilities_concept.md) V3.0 flag must be set to true.
+
+### Local configuration
+
+The `orderer.yaml` has two configuration sections that are relevant for BFT
+orderers:
+
+**Cluster**, which determines the communication configuration, and
+**consensus**, which determines BFT protocol related configuration.
+
+**Cluster parameters:**
+
+By default, the BFT service is running in the same gRPC server as the client
+facing gRPC API (which is used for transaction submission or block retrieval), but it can be
+configured to have a separate gRPC server with a separate port.
+
+This is useful for cases where you want TLS certificates issued by the
+organizational CAs, but used only by the cluster nodes to communicate among each
+other, and TLS certificates issued by a public TLS CA for the client facing API.
+
+* `ClientCertificate`, `ClientPrivateKey`: The file path of the client TLS certificate
+  and corresponding private key.
+* `ListenPort`: The port the cluster listens on.
+  It must be same as `consenters[i].Port` in Channel configuration.
+  If blank, the port is the same port as the orderer general port (`general.listenPort`)
+* `ListenAddress`: The address the cluster service is listening on.
+* `ServerCertificate`, `ServerPrivateKey`: The TLS server certificate key pair
+  which is used when the cluster service is running on a separate gRPC server
+  (different port).
+
+Note: `ListenPort`, `ListenAddress`, `ServerCertificate`, `ServerPrivateKey` must
+be either set together or unset together.
+If they are unset, they are inherited from the general TLS section,
+in example `general.tls.{privateKey, certificate}`.
+When general TLS is disabled:
+- Use a different `ListenPort` than the orderer general port
+- Properly configure TLS root CAs in the channel configuration.
+
+There are also hidden configuration parameters for `general.cluster` which can be
+used to further fine tune the cluster communication or replication mechanisms:
```

Comment:
Are these all really identical to the Raft config? If so, nice job making it consistent!

Reply:
There is no change in the cluster configuration. The only change is in the authentication mechanism which is not configurable anyway.